aiida/cmdline/commands/cmd_daemon.py

```diff
@@ -211,7 +211,7 @@ def restart(ctx, reset, no_wait):
 @click.option('--foreground', is_flag=True, help='Run in foreground.')
 @decorators.with_dbenv()
 @decorators.check_circus_zmq_version
-def start_circus(foreground):
+def start__pcircus(foreground):
```

Comment:
why the name change? Does this accomplish the hiding of the command? I found that `click==7.0.0` has implemented a `hidden` keyword for `commands` which works, I just couldn't get the bash completion to work though. It is also supposed to hide it from autocompletion. In any case, I would leave this untouched and do a proper click based solution when we get it to work

Reply:
This somehow popped out of the merge - it's not like I wanted to add this.
Will revert the name